common/source/docs/common-landing-gear.rst

```diff
@@ -0,0 +1,39 @@
+.. _common-landing-gear:
+
+======================================
+Landing Gear/ Retractable Camera Mount
+======================================
+
+ArduCopter and ArduPlane support retractable Landing Gear / Camera Mounts activated by a servo mechanism(like these sold by `Hobby King <https://hobbyking.com/en_us/quanum-600-class-quick-release-universal-retractable-gear-set-680uc-pro-hexa-copter.html?___store=en_us>`__ for copters).
```

Comment:
I think we should consolidate on just saying "Copter", "Plane" etc instead of "ArduCopter" and "ArduPlane"